src/app/core/models/issue.model.ts

```diff
@@ -126,9 +126,9 @@ export class Issue {
     issue.githubComments = githubIssue.comments;
     issue.teamAssigned = teamData;
     issue.issueComment = template.comment;
-    issue.teamResponse = template.teamResponse !== undefined ? Issue.updateTeamResponse(template.teamResponse.content) : undefined;
-    issue.duplicateOf = template.duplicateOf !== undefined ? template.duplicateOf.issueNumber : undefined;
-    issue.duplicated = issue.duplicateOf !== undefined && issue.duplicateOf !== null;
+    issue.teamResponse = template.teamResponse && Issue.updateTeamResponse(template.teamResponse.content);
+    issue.duplicateOf = template.duplicateOf && template.duplicateOf.issueNumber;
+    issue.duplicated = !!issue.duplicateOf;
```

Comment:
This statement would set `issue.duplicated` to false, in the case when `issue.duplicateOf` is `0`.
This is quite an unlikely edge case, as currently GitHub starts issue numbers from 1.
But I still think it's better to keep the original statement, rather than accepting the buggy edge case.

Reply:
@anubh-v 
Yes, I've thought about that case as well. What makes me from the past eventually go with `!!issue.duplicateOf` is the fact that GitHub issues/PRs are numbered starting from 1.

But yeah, to be very safe I think reverting back to the original statement is a good choice. Thanks for the input!